x-pack/plugins/security_solution/public/common/components/header_section/index.tsx

```diff
@@ -108,10 +128,16 @@ const HeaderSectionComponent: React.FC<HeaderSectionProps> = ({
       className={classNames}
       $hideSubtitle={hideSubtitle}
     >
-      <EuiFlexGroup direction="column" responsive={false} gutterSize="xs">
-        <EuiFlexItem>
+      <EuiFlexGroup
+        data-test-subj="headerSectionOuterFlexGroup"
+        direction={outerDirection}
+        gutterSize="xs"
+        responsive={false}
+      >
+        <EuiFlexItem grow={growLeftSplit}>
           <EuiFlexGroup
-            alignItems={stackHeader ? undefined : 'center'}
+            alignItems={getHeaderAlignment({ alignHeader, stackHeader })}
```

Comment:
`useMemo`?

Reply:
> `useMemo`?

In this case it shouldn't be required, because `getHeaderAlignment` is a pure function with the following return type:

```ts
"center" | "baseline" | "stretch" | "flexStart" | "flexEnd" | undefined
```